frontend/public/components/utils/details-page.tsx

```diff
@@ -49,14 +49,14 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({
   showAnnotations = true,
   showTolerations = false,
   podSelector = 'spec.selector',
+  nodeSelector = '',
 }) => {
   const { metadata, type } = resource;
   const reference = referenceFor(resource);
   const model = modelFor(reference);
-  const nodeSelectorPath = getNodeSelectorPath(resource);
+  const nodeSelectorPath = _.isEmpty(nodeSelector) ? getNodeSelectorPath(resource) : nodeSelector;
```

Comment:
Remove `getNodeSelectorPath` and pass it in for the one other exception we have for pods.

Reply:
@spadgett fixed as you suggested.  Wish there was a way to search and be certain that there was only 'one other [Pod] exception'.  I see some `<ResourceSummary ../>`'s use `item.obj`:
![image](https://user-images.githubusercontent.com/12733153/78620505-2dd5f400-784e-11ea-9ef6-a7abe0d5512d.png)
These are used by TopologyResourcePanel.tsx. Clicking around Topology viewer I can't bring up any Pod objects, but not 100% sure one can't.

Reply:
Hi, just confirmed with @jeff-phillips-18 that Pods aren't items shown in the topology view.  So passing `item.obj` to `<ResourceSummary />` on the topology side panel is ok and we don't have to pass a `nodeSelector` path for Pods.
![image](https://user-images.githubusercontent.com/12733153/78683848-cc4d6e00-78bd-11ea-81aa-bfd3f0ed6043.png)